<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameSetting</class>
 <widget class="QDialog" name="GameSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="gameStart">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>280</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>227</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="p2Name">
    <item>
     <widget class="QLabel" name="label_p2Name">
      <property name="text">
       <string>Player2's name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_p2Name">
      <property name="text">
       <string>Player2</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>227</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="targetScore">
    <item>
     <widget class="QLabel" name="label_targetScore">
      <property name="text">
       <string>Target score: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_targetScore">
      <property name="text">
       <string>20</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>227</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="p1Name">
    <item>
     <widget class="QLabel" name="label_p1Name">
      <property name="text">
       <string>Player1's name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_p1Name">
      <property name="text">
       <string>Player1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
